package.json

```diff
@@ -61,5 +61,6 @@
     "engines": {
         "npm": "please-use-yarn",
         "yarn": ">= 1.19.1"
-    }
+    },
+    "proxy": "http://yerkee.com/api"
```

Comment:
Does it work on deploy? I don't think so, the real solution is to allow for cross origin access in axios request header